modules/i18n/src/translations/en-US/portals/organizations.ts

```diff
@@ -129,6 +134,17 @@ export const organizations: organizationsNS = {
                     empty: "Organization name is required"
                 }
             },
+            orgHandle: {
+                label: "Organization Handle",
+                placeholder: "Organization handle (E.g., myorg.com)",
+                validation: {
+                    duplicate: "The Organization handle already exists",
+                    empty: "The Organization handle is required",
+                    invalidPattern: "The Organization handle contains invalid characters. The valid characters are letters, numbers, spaces, '.', '-', and '_'.",
+                    mandatoryExtension: "The Organization handle should have a dot extension. E.g,: abc.com",
```

Comment:
the term "dot extension" is not much common, is it? how about "domain extension"?.

```suggestion
                    mandatoryExtension: "The organization handle should include a domain extension, such as .com. (E.g,: abc.com)",
```

Reply:
This is now updated.